ci-operator/config/integr8ly/ansible-tower-configuration/integr8ly-ansible-tower-configuration-master.yaml

```diff
@@ -0,0 +1,26 @@
+base_images:
+  os:
+    name: centos
+    namespace: openshift
+    tag: '7'
+
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      memory: 250Mi
+      cpu: 100m
+  unit:
+    limits:
+      memory: 10Gi
+    requests:
+      memory: 2Gi
+      cpu: 200m
+
+
+tests:
```

Comment:
I'm just adding this one for now to test the ci.
We'll add other test later as part of the follow on ticket. 
Building jobs requires at least one of an image build or test target.